retirement/models.py

```diff
@@ -1317,10 +1317,17 @@ def send_refund_confirmation_email(email_dict):
         user = email_dict['user']
         total_amount = email_dict['total_amount']
         amount_tax = email_dict['amount_tax']
+        refund_policy = email_dict['refund_policy']
         # Here the price takes the applied coupon into account, if
         # applicable.
+        if refund_policy == 'full_rate':
+            price = amount
+        elif refund_policy == 'retreat_rate':
+            price = (amount * retreat.refund_rate) / 100
+        else:
+            price = 0  # no refund
```

Comment:
It's really dangerous to have a `default` condition when we are speaking with these amount of money. Don't we have a way to force all conditions to a specific value ?